tdh/data_source_data_planes.go

```diff
@@ -201,7 +201,7 @@ func (d *dataPlaneDatasource) Schema(_ context.Context, _ datasource.SchemaReque
 // Read refreshes the Terraform state with the latest data.
 func (d *dataPlaneDatasource) Read(ctx context.Context, req datasource.ReadRequest, resp *datasource.ReadResponse) {
 	var state dataPlaneDatasourceModel
-	var dataPlaneList []dataPlaneModel
```

Comment:
this model is only for single representation, omit the "s" from "DataPlanesModel"